samples/ronn/bunyan.1.ronn

```diff
@@ -0,0 +1,195 @@
+# bunyan(1) -- filter and pretty-print Bunyan log file content

```

Comment:
The sample files should be in `samples/Markdown/`. (It's the cause of the of the Travis CI error.)

Reply:
fixed 👍